### ISSUE TITLE
Ctrl+X doesn't work.

### ISSUE THREAD

**Issue by m00nwtchr** [lapce/lapce#1652]
## Lapce Version
v0.2.2 (b843959ac10a23ef6ba9f75be65586e24685dac3, `lapce-git` AUR package)

## System information

Arch Linux (Wayland, dwl)

## Describe the bug
Ctrl+X doesn't seem to be recognized at all by the application, actions bound to it don't trigger, and I can't bind an action to Ctrl+X.

## Additional information

Other information that can be used to further reproduce or isolate the problem.
This commonly includes:

- screenshots
- logs
- theories about what might be going wrong
- workarounds that you used

**Comment by NicolasTriquenaux** [comment]
I have fallen into that bug, restarting Lapce solved the issue.

**Comment by m00nwtchr** [comment]
Yeah restarting solves it for me as well, forgot to mention that. I'll see if it's correlated /w that menu as well (I mainly code in Rust)

**Comment by MatthewScholefield** [comment]
I also encountered the same thing after messing with the lightbulb with the python lsp plugin (unrelated, did those work for you? Couldn't get it to work even with `pylsp-rope` installed)

**Comment by ethereumdegen** [comment]
Oddly enough,  shift+delete still works for me when this happens.  but it keeps happening to me and its soo annoying

**Comment by alexpyattaev** [comment]
+1 to the issue suffering set. Is everyone that have this issue using Arch linux by some chance? Could it be arch-specific bug?

**Comment by Goddard** [comment]
same manjaro
